scripts/dependencies/docs/Pipfile.lock

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
We don't maintain a separate `Pipfile.lock` for the docs; so this file is not needed. It is purposely removed via this relocking of dependencies script which handles the repos dependencies - https://github.com/nucypher/nucypher/blob/main/scripts/dependencies/relock_dependencies.sh#L39.

The generated `docs-requirements.txt` file is sufficient for docs dependencies, and is used by ReadTheDocs when generating the documentation (cc @KPrasch).